ledger/participant-integration-api/src/main/scala/platform/configuration/SubmissionConfiguration.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.configuration
+
+/** Configuration of the Ledger API Command Submission Service
+  * @param enableDeduplication
+  *        Specifies whether the participant deduplication can be turned on or off.
+  *        By default, deduplication is turned on. There is a class of ledgers where the
+  *        deduplication is implemented by the committer. On those ledgers this parameter
+  *        has no impact.
```

Comment:
It may be worth being a bit more specific here to avoid ambiguity:
```suggestion
  *        By default, command deduplication is turned on. However, on ledgers where deduplication
  *        is implemented by the committer this parameter has no impact.
```

Reply:
Done